CSharp/Library/Microsoft.Bot.Builder/Dialogs/LuisDialog.cs

```diff
@@ -140,6 +143,10 @@ public LuisServiceResult(LuisResult result, IntentRecommendation intent, ILuisSe
     [Serializable]
     public class LuisDialog<TResult> : IDialog<TResult>
     {
+        public const string LuisTraceType = "https://www.luis.ai/schemas/trace";
+        public const string LuisTraceLabel = "Luis Trace";
+        public const string Obfuscated = "****";
```

Comment:
>Obfuscated [](start = 28, length = 10)

????

Reply:
Added it but forgot to use it, fixed, thanks!